### PR TITLE
Update versions for August release

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.1.0-preview.5 (Unreleased)
+## 0.1.0-preview.5 (2020-09-03)
 
 ### Added
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:5]: We added some breaking changes for BinaryData.

Comment (pakrym) [reply sdk/core/Azure.Core.Experimental/CHANGELOG.md:5]: Want to push an update to this branch?

Comment (JoshLove-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:5]:
```suggestion
### Added
```
```suggestion
### Added
- `JsonPatchDocument` type to represent JSON Path document.
- `BinaryData`: FromString method.
- `BinaryData`: FromBytes method taking ReadOnlySpan.
- `BinaryData`: constructor taking ReadOnlyMemory.

### Breaking Changes
- `BinaryData`: Renamed `Serialize` to `FromObject`.
- `BinaryData`: Renamed `Deserialize` to `ToObject`.
- `BinaryData`: Renamed `FromMemory` to `FromBytes`.
```

Comment (JoshLove-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:5]:
```suggestion
### Added
- `JsonPatchDocument` type to represent JSON Path document.
- `BinaryData`: FromString method.
- `BinaryData`: FromBytes method taking ReadOnlySpan.
- `BinaryData`: constructor taking ReadOnlyMemory.

### Breaking Changes
- `BinaryData`: Renamed `Serialize` to `FromObject`.
- `BinaryData`: Renamed `Deserialize` to `ToObject`.
- `BinaryData`: Renamed `FromMemory` to `FromBytes`.
```

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
 - `JsonPatchDocument` type to represent JSON Path document.
+- `BinaryData`: FromString method.
+- `BinaryData`: FromBytes method taking ReadOnlySpan.
+- `BinaryData`: constructor taking ReadOnlyMemory.
+
+### Breaking Changes
+- `BinaryData`: Renamed `Serialize` to `FromObject`.
+- `BinaryData`: Renamed `Deserialize` to `ToObject`.
+- `BinaryData`: Renamed `FromMemory` to `FromBytes`.
+- `JsonPatchDocument` type to represent JSON Path document.
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:15]:
```suggestion
```

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**